plugin-dev/Source/Sentry/Private/Android/SentrySubsystemAndroid.cpp

```diff
@@ -169,6 +169,32 @@ USentryId* SentrySubsystemAndroid::CaptureEventWithScope(USentryEvent* event, co
 	return SentryConvertorsAndroid::SentryIdToUnreal(*id);
 }
 
+USentryId* SentrySubsystemAndroid::CaptureException(const FString& type, const FString& message, int32 framesToSkip)
+{
+	// add marker tags specific for Unreal assertions
+	SetTag(TEXT("sentry_unreal_exception"), TEXT("assert"));
+	SetTag(TEXT("sentry_unreal_exception_skip_frames"), FString::Printf(TEXT("%d"), framesToSkip));
+	SetTag(TEXT("sentry_unreal_exception_type"), type);
+	SetTag(TEXT("sentry_unreal_exception_message"), message);
+
+	PLATFORM_BREAK();
+
+	return nullptr;
+}
+
+USentryId* SentrySubsystemAndroid::CaptureAssertion(const FString& type, const FString& message)
+{
+	return CaptureException(type, message, 8);
```

Comment:
Alright, so assertions and exceptions are getting treated similarly. Why the hardcoded `8`? Is that the same for all UE versions on all Android versions?